provisioning_client/tests/common_prov_e2e/prov_hsm/x509_msr.c

```diff
@@ -72,6 +72,9 @@ static char g_device_name[64] = { 0 };
 
 static int g_digest_initialized = 0;
 
+// Free the mbedtls_mpi members of the signature
+static void ecc_signature_destroy(RIOT_ECC_SIGNATURE* tbs_sig);
```

Comment:
Same as above - moving the definition here is going to be easier to read and have less number of code lines.

Reply:
Fixed

Reply:
Please push the change (and eventually resolve the conversation).